deploy/configmap.yaml

```diff
@@ -5,3 +5,5 @@ metadata:
   namespace: ingress-nginx
   labels:
     app: ingress-nginx
+
+---
```

Comment:
[According to the YAML spec](http://yaml.org/spec/1.2/spec.html#id2760395), this marks the beginning of a document, not the end.

Reply:
done